app/models/criteria/uncompleted_task.rb

```diff
@@ -16,6 +17,12 @@ def self.create(searcher_id, assignee_id: nil, from_date: nil, to_date: nil, tag
             plans.add(task_deadline: :left_outer)
           end
         end
+
+        c.set_sorter do |result_set|
+          with_deadline = result_set.select(&:deadline).sort_by { |r| r.deadline }
+          without_deadline = (result_set - with_deadline).sort_by { |r| r.id }
```

Comment:
__[rubocop]__
* Style/SymbolProc: Pass `&:id` as an argument to `sort_by` instead of a block.